javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $currentBG = document.querySelector('#currentBG');
+var $currentCircle = document.querySelector('#currentCircle');
+
+function doSwitch(event) {
+  if ($currentBG.className === 'lightBG') {
+    $currentBG.className = 'darkBG';
+    $currentCircle.className = 'circle darkCircle';
+  } else {
+    $currentBG.className = 'lightBG';
+    $currentCircle.className = 'circle lightCircle';
+  }
+}
+
+$currentCircle.addEventListener('click', doSwitch, false);
```

Comment:
You don't need to pass that boolean as an argument to `addEventListener`, it is the default behavior, so it is unnecessary 🤓